lib/extensions/ar_migration.rb

```diff
@@ -69,9 +69,11 @@ def wait_message
     end
 
     def restart_subscription
-      c = SubscriptionHelper.establish_connection.connection
-      c.pglogical.subscription_disable(subscription.id)
-      c.pglogical.subscription_enable(subscription.id)
+      c = SubscriptionHelper.establish_connection.connection.raw_connection
+      rep_client = PG::LogicalReplication::Client.new(c)
+
+      rep_client.disable_subscription(subscription.id)
+      rep_client.enable_subscription(subscription.id)
```

Comment:
Minor, but we can probably create a `.restart_subscription` method  right in the client, that can do the disable/enable pair

Reply:
I don't think it's that common a use case. I actually debated just removing this bit because it retries the subscription every 5 seconds, but opted to reduce the amount of change after testing that this still works.